definitions/webbrowser.yaml

```diff
@@ -273,6 +273,7 @@ sources:
     paths:
       - '%%users.localappdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
       - '%%users.appdata%%\Mozilla\Firefox\Profiles\*\places.sqlite'
+      - '%%users.appdata%%\Roaming\Mozilla\Firefox\Profiles\*\places.sqlite'
```

Comment:
%APPDATA% should already contain Roaming
also see: https://en.wikipedia.org/wiki/Environment_variable